packs/aws.yml

```diff
@@ -0,0 +1,66 @@
+AnalysisType: pack
+PackID: PantherManaged.AWS.Core
+Description: Group of the most critical and high value detections pertinent to the aws environment
+PackDefinition:
+  IDs:
+    # Data Exposure
+    - AWS.AMI.Private
+    - AWS.CloudTrail.AMIModifiedForPublicAccess
+    - AWS.CloudTrail.S3Bucket.Public
+    - AWS.IAM.AccessKeyCompromised
+    - AWS.KMS.RestrictsUsage
+    - AWS.RDS.Instance.PublicAccess
+    - AWS.RDS.Instance.SnapshotPublicAccess
+    - AWS.S3.Bucket.PublicRead
+    - AWS.S3.Bucket.PublicWrite
+    - AWS.S3.Bucket.PolicyAllowWithNotPrincipal
+    - AWS.S3.Bucket.PrincipalRestrictions
+    # Encryption Status
+    - AWS.DynamoDB.TableEncryption
+    - AWS.EC2.Volume.Encryption
+    - AWS.EC2.Volume.Snapshot.Encrypted
+    - AWS.Redshift.Cluster.Encryption
+    - AWS.RDS.Instance.Encryption
+    - AWS.S3.Bucket.Encryption
+    # Networking Policies
+    - AWS.NetworkACL.RestrictsInboundTraffic
+    - AWS.SecurityGroup.AdministrativeIngress
+    - AWS.SecurityGroup.OnlyDMZPubliclyAccessible
+    - AWS.SecurityGroup.RestrictsInboundTraffic
```

Comment:
```suggestion
    # Networking Policies
    - AWS.NetworkACL.RestrictsInboundTraffic
    - AWS.SecurityGroup.AdministrativeIngress
    - AWS.SecurityGroup.OnlyDMZPubliclyAccessible
    - AWS.SecurityGroup.RestrictsAccessToCDE
    - AWS.SecurityGroup.RestrictsInboundTraffic
```

Reply:
Nice find! I was considering it, but the policy body mentions:

```
# This is a generic policy that checks inbound permissions on a Security Group.
# You may wish to add additional logic specific to your use cases.
```

which to me seems a bit between "configuration required" and not needing configuration. I'm on the fence about it - so if you have a strong opinion that it should be included I can do that :)